website/docs/reference/resource-configs/spark-configs.md

```diff
@@ -320,3 +391,28 @@ snapshots:
 ```
 
 </File>
+
+
+## Apache Hudi configurations
```

Comment:
Same note as above, I feel as if this section can be generalized to avoid duplication.

Reply:
good point, updated it.